meteor/override-fastclick.js

```diff
@@ -0,0 +1,6 @@
+// disable fastclick package in Meteor if included
```

Comment:
Do you mean inside the fastclick package?
My guess is that probably not and if so, it might take a while (like next version or something)
The good thing about that is here it's out of both of the platforms specific code and only in the integrating package